client/mainwindow.cpp

```diff
@@ -465,6 +540,36 @@ bool MainWindow::saveAccessToken(const AccountSettings& account,
     return false;
 }
 
+#ifdef USE_KEYCHAIN
+bool MainWindow::saveAccessTokenToKeyChain(const QMatrixClient::AccountSettings &account,
```

Comment:
```suggestion
bool MainWindow::saveAccessTokenToKeyChain(const AccountSettings& account,
```
`QMatrixClient::AccountSettings` is under `using` throughout the file; also aligning ampersand positioning with the rest of the codebase.

Reply:
done